centaurCwlRunner/src/main/scala/centaur/cwl/OutputManipulator.scala

```diff
@@ -144,6 +144,17 @@ object OutputManipulator extends Poly1 {
       case (JsNumber(metadata), Inl(CwlType.Double)) => metadata.doubleValue.asJson
       case (JsNumber(metadata), Inl(CwlType.Int)) => metadata.intValue.asJson
       case (JsString(metadata), Inl(CwlType.String)) => metadata.asJson
+
+      //The Anys.  They have to be done for each type so that the asJson can use this type information when going back to Json representation
+      case (JsString(metadata), Inl(CwlType.Any)) => metadata.asJson
+      case (JsNumber(metadata), Inl(CwlType.Any)) => metadata.asJson
+      case (obj: JsObject, Inl(CwlType.Any)) =>
+        import io.circe.parser._
+        parse(obj.compactPrint).right.getOrElse(throw new Exception("Failed to parse Json output as Json... something is very wrong"))
+      case (JsBoolean(metadata), Inl(CwlType.Any)) => metadata.asJson
+      case (JsArray(metadata), Inl(CwlType.Any)) => metadata.asJson
```

Comment:
it seems like most of these just `unapply` from specific `JsFoo` types and then re-JSONify them? I think the comment is trying to tell me otherwise but I'm not getting it... 😢

Reply:
If you stare too closely at the ~sun~ codebase you'll find a number of places with "SprayJson <-> Circe" conversions like this. In this case it's because of `case class centaur.test.metadata.WorkflowMetadata(value: Map[String, JsValue])`.

New code like this OutputManipulator has been using Circe, while we still have SprayJson sprinkled inside classes all over. Someday perhaps a SOC refactoring should pull the SprayJson _and_ Circe code out into wrappers, otherwise I foresee "SprayJson <-> Circe <-> (insert next scala-json hotness here)" conversions in our future.

Reply:
Or someone does this ticket :)

https://github.com/broadinstitute/cromwell/issues/2423